manifests/03-rbac-role-ns-openshift-config.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator
+  namespace: openshift-config
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - secrets
```

Comment:
We don't want `secrets`, just `configmaps`.

Reply:
The deafult code for ResourceSync expects a secretsGetter and that getter will only work with RBAC to the namespaces.  I am working on a way around this.